resources.qrc

```diff
@@ -1,9 +1,11 @@
 <RCC>
   <qresource prefix="/">
+    <file>images/pause.png</file>
     <file>images/power_off.png</file>
     <file>images/back.png</file>
     <file>images/forward.png</file>
     <file>images/play.png</file>
```

Comment:
Oops sorry, apparently both added by hand and via designer... should only be once...
Will remove it and squash to one commit. Hopefully adding svg images to this PR too